### DISCUSSION TITLE
Does Vector Support Kubernetes Metadata ?

### DISCUSSION THREAD

**Discussion by hamedsa-78** [vectordotdev/vector#21355] [Q&A] [answered]
Does Vector have a Kubernetes metadata plugin like Fluentd? I am not referring to Vector's Kubernetes logs in the source section, as it doesn't support custom pod selection. Additionally, I need it for arbitrary log paths of a pod, not just the default.

**Comment by jszwedko** [comment, marked as answer]
There isn't a separate transform, [yet](https://github.com/vectordotdev/vector/issues/5077), but the `kubernetes_logs` source does add k8s metadata to the logs it reads.
